packages/devtools_app/lib/src/flutter/table.dart

```diff
@@ -418,6 +453,22 @@ class _TableRow<T> extends StatefulWidget {
 
 class _TableRowState<T> extends State<_TableRow<T>>
     with TickerProviderStateMixin, CollapsibleAnimationMixin {
+  Key contentKey;
+
+  @override
+  void initState() {
+    super.initState();
+    contentKey = ValueKey(this);
+    expandController.addStatusListener((status) {
```

Comment:
does this need to be disposed?

Reply:
The collapsible animation mixin creates the expandController, and it will handle disposing it.